src/components/FindMovie/FindMovie.tsx

```diff
@@ -17,41 +81,56 @@ export const FindMovie: React.FC = () => {
               id="movie-title"
               placeholder="Enter a title to search"
               className="input is-dander"
+              value={searchValue}
+              onChange={handleSearchInput}
             />
           </div>
 
-          <p className="help is-danger" data-cy="errorMessage">
-            Can&apos;t find a movie with such a title
-          </p>
+          {error
+          && (
+            <p className="help is-danger" data-cy="errorMessage">
+              Can&apos;t find a movie with such a title
+            </p>
+          )}
         </div>
 
         <div className="field is-grouped">
           <div className="control">
             <button
               data-cy="searchButton"
               type="submit"
-              className="button is-light"
+              className={classNames(
+                'button',
+                'is-light',
+                { 'is-loading': isLoading },
+              )}
+              disabled={!searchValue.length}
             >
-              Find a movie
+              {firstFound ? 'Search again' : 'Find a movie'}
```

Comment:
```suggestion
              {foundMovie ? 'Search again' : 'Find a movie'}
```
you can use here foundMovie and get rid of firstFound variable in your code